node.go

```diff
@@ -141,3 +143,49 @@ func (n *Node) References(refs *ua.NodeID) (*ua.BrowseResponse, error) {
 	return n.c.Browse(req)
 	// implement browse_next
 }
+
+//translate an array of browseName segment to NodeID
+func (n *Node) TranslateBrowsePathInMultiNamespaceToNodeId(pathNames []*ua.QualifiedName) (nodeId *ua.NodeID, err error) {
+	req := ua.TranslateBrowsePathsToNodeIDsRequest{
+		RequestHeader: &ua.RequestHeader{AuthenticationToken: ua.NewFourByteNodeID(0, id.TranslateBrowsePathsToNodeIDsRequest_Encoding_DefaultBinary)},
+		BrowsePaths: []*ua.BrowsePath{
+			{StartingNode: n.ID, RelativePath: &ua.RelativePath{Elements: []*ua.RelativePathElement{}}},
+		}}
+
+	for _, pathSegment := range pathNames {
+		req.BrowsePaths[0].RelativePath.Elements = append(req.BrowsePaths[0].RelativePath.Elements,
+			&ua.RelativePathElement{ReferenceTypeID: ua.NewTwoByteNodeID(id.HierarchicalReferences),
+				IsInverse:       false,
+				IncludeSubtypes: true,
+				TargetName:      pathSegment},
+		)
+	}
+
+	err = n.c.Send(&req, func(i interface{}) error {
+		if resp, ok := i.(*ua.TranslateBrowsePathsToNodeIDsResponse); ok {
+			if len(resp.Results) > 0 {
+				if resp.Results[0].StatusCode == ua.StatusOK {
+					if len(resp.Results[0].Targets) > 0 {
+						nodeId = resp.Results[0].Targets[0].TargetID.NodeID
+						return nil
+					}
+				} else {
+					return errors.New(resp.Results[0].StatusCode.Error())
+				}
+			}
+		}
+		return errors.New("unknown error")
```

Comment:
I recall having [this discussion](https://github.com/gopcua/opcua/pull/188#discussion_r282367121) about error codes.
> If we don't use status codes then the error message becomes part of the public API and users will have to check for that. Lets stick to status codes unless there is a good reason not to.
Maybe you should stick to status codes  here as well.
The same applies to line 173.